src/plugins/exportAsJSONAction/ExportAsJSONAction.js

```diff
@@ -59,7 +57,11 @@ export default class ExportAsJSONAction {
      * @param {object} objectpath
      */
     invoke(objectpath) {
+        this.externalIdentifiers = [];
         this.tree = {};
+        this.calls = 0;
+        this.idMap = {};
```

Comment:
🥇

Reply:
haha, thanks for pointing this out @shefalijoshi !